src/stubs/cssVar.js

```diff
@@ -41,4 +41,4 @@ function cssVar(name) {
 
 cssVar.CSS_VARS = CSS_VARS;
 
-export default cssVar;
+module.exports = cssVar;
```

Comment:
one of our custom loaders import this file

the custom loader doesn't recognize es6 module style

Reply:
which custom loader?

Reply:
> which custom loader?

[cssTransformLoader](https://github.com/schrodinger/fixed-data-table-2/blob/master/build_helpers/cssTransformLoader.js#L6)
It uses cssVar to get the variable definitions for the ones used in our CSS

Reply:
Interesting, I've never seen these before :(